beacon-chain/rpc/beaconv1/pool.go

```diff
@@ -56,3 +78,24 @@ func (bs *Server) ListPoolVoluntaryExits(ctx context.Context, req *ptypes.Empty)
 func (bs *Server) SubmitVoluntaryExit(ctx context.Context, req *ethpb.SignedVoluntaryExit) (*ptypes.Empty, error) {
 	return nil, errors.New("unimplemented")
 }
+
+func mapAttestation(attestation *eth.IndexedAttestation) *ethpb.IndexedAttestation {
```

Comment:
What does `mapAttestation` mean? Do you just mean copy?

Reply:
These are different types - `*eth` and `*ethpb`. It's a mapping from old proto to new proto. I will change the name to make it more clear.

Reply:
Sorry i missed the prefix. This also seems useful, perhaps a better file location in `pool` or even `shared`

Reply:
This is a temporary translation between the v1alpha1 and v1 objects. A comment would be helpful here.

Reply:
In https://github.com/prysmaticlabs/prysm/pull/7525/ I made a migration package for utils like this, maybe that would be useful here?